### PR TITLE
Update QrCode package

### PR DESCRIPTION
**Pull request by SimplyCorey**
Quick update to the QrCode package in composer.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "require": {
         "php": ">=5.3.7",
         "christian-riesen/base32": "~1.0",
-        "simplesoftwareio/simple-qrcode" : "1.3.*"
+        "simplesoftwareio/simple-qrcode" : "1.4.*"
```
Comment (aik099) [inline composer.json]: Please change this into `^1.3`. This would all all versions from 1.3 till, not including 2.0.

**Comment by SimplyCorey** [comment]
@aik099 Good to go

**Comment by GrahamCampbell** [comment]
Does `simplesoftwareio/simple-qrcode` follow semver? Is this change safe to do?

**Comment by SimplyCorey** [comment]
They follow semver and there is no breaking changes in 1.4.

**Comment by GrahamCampbell** [comment]
What about in earlier versions? We need to be sure 1.5, or 1.6 won't break, etc.

**Comment by SimplyCorey** [comment]
It has always followed semver.  Would it be better to switch this back to `1.4.*`?  The next version is going to be `2.0` which is going to have some backwards breaking changes.  `1.5` and above will never exist.

**Comment by GrahamCampbell** [comment]
If they are semver, `^1.3` is just fine.

**Comment by barryvdh** [comment]
Or drop this package in favor of the underlying package; https://github.com/Bacon/BaconQrCode
See #36

**Comment by antonioribeiro** [comment]
Closing it because we'' ll move to BaconQrCode.
Thank you, anyway ;)

**Pull request closed without merging**